src/cascadia/TerminalControl/TSFInputControl.cpp

```diff
@@ -83,11 +71,8 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     // - <none>
     void TSFInputControl::NotifyFocusEnter()
     {
-        if (_editContext != nullptr)
-        {
-            _editContext.NotifyFocusEnter();
-            _focused = true;
-        }
+        _editContext.NotifyFocusEnter();
+        _focused = true;

```

Comment:
Nothing in this class sets `_editContext` to `nullptr`. 🤔

Reply:
Hmm. Good point. We'll totally blow up in the ctor long before we get here.